nsv13/code/__HELPERS/names.dm

```diff
@@ -38,3 +38,14 @@ var/list/ship_name_suffix = list("slayer","blazer","bringer","slapper","twirler"
 	var/name = ""
 	name = pick(ship_name_prefix) + pick(ship_name_suffix)
 	return capitalize(name)
+
+/proc/generate_fighter_name()
+	var/name = ""
+	var/randomizer = rand(0,1)
+	var/chooser = FALSE
+	if(randomizer == 0)
+		chooser = FALSE
+	else
+		chooser = TRUE
```

Comment:
1 and 0 are already TRUE and FALSE in byond, making randomizer and chooser the same (if one is 1 it is also TRUE and vice versa). The next comment contains the preferred solution.